app/models/vm.rb

```diff
@@ -1,6 +1,7 @@
 class Vm < VmOrTemplate
   default_scope { where(:template => false) }
-
+  has_one :container_deployment, :through => :deployments_node_deployment
+  has_one :container_node_deployment
```

Comment:
@alongoldboim same as above, not sure whether this belongs here, we need an ack form @blomquisg @gmcculloug @Fryguy